File: module22/module01/module43/src/main/java/com/android/example/module22_module01_module43/ClassAAE.java
package com.android.example.module22_module01_module43;

public class ClassAAE {
	private 	com.android.example.module01_module39.ClassAAH instance_var_1_0 = new com.android.example.module01_module39.ClassAAH();
	private 	com.android.example.module01_module39.ClassAAB instance_var_1_1 = new com.android.example.module01_module39.ClassAAB();
	private 	com.android.example.module01_module39.ClassAAJ instance_var_1_2 = new com.android.example.module01_module39.ClassAAJ();
	private 	com.android.example.module01_module39.ClassAAC instance_var_1_3 = new com.android.example.module01_module39.ClassAAC();

	public void method0(
		com.android.example.module01_module39.ClassAAF param0,
		com.android.example.module01_module39.ClassAAB param1,
		com.android.example.module01_module39.ClassAAG param2) throws Throwable {
	}

	public void method1(
		com.android.example.module01_module39.ClassAAH param0,
		com.android.example.module01_module39.ClassAAE param1) throws Throwable {
		java.util.Collections.emptyList().forEach( lambda0 -> {
			try {
				io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
				local_var_3_0.requestMore(-1195927213l);

				local_var_3_0.requestMore(-1501611765l);

			} catch(Throwable e) { } // ignore
		});
		param0.method0(new dagger.internal.DelegateFactory());

		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_2 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_2.isDisposed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.assertNotSubscribed();

	}

	public void method2(
		com.android.example.module01_module39.ClassAAB param0,
		com.android.example.module01_module39.ClassAAJ param1) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.assertSubscribed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.isCancelled();

		io.reactivex.observers.TestObserver<Object> local_var_2_4 = new io.reactivex.observers.TestObserver();
		local_var_2_4.isCancelled();

	}
}

File: module18/module1/module24/module2/src/main/java/com/android/example/module18_module1_module24_module2/ClassAAE.kt
package com.android.example.module18_module1_module24_module2

@Suppress("UNUSED_PARAMETER")
class ClassAAE {
	private 	val instance_var_1_0: com.android.example.module18_module1_module24_module1.ClassAAF = com.android.example.module18_module1_module24_module1.ClassAAF()
	private 	val instance_var_1_1: com.android.example.module18_module1_module24_module1.ClassAAG = com.android.example.module18_module1_module24_module1.ClassAAG()
	private 	val instance_var_1_2: com.android.example.module18_module1_module24_module3.ClassAAC = com.android.example.module18_module1_module24_module3.ClassAAC()
	private 	val instance_var_1_3: com.android.example.module18_module1_module24_module1.ClassAAB = com.android.example.module18_module1_module24_module1.ClassAAB()

	fun method0(
		param0: com.android.example.module18_module1_module24_module3.ClassAAD,
		param1: com.android.example.module18_module1_module24_module1.ClassAAI,
		param2: com.android.example.module18_module1_module24_module3.ClassAAD
	) {
	}

	fun method1(
		param0: com.android.example.module18_module1_module24_module3.ClassAAE,
		param1: com.android.example.module18_module1_module24_module3.ClassAAG
	) {
		listOf("1", "2", "3").forEach {
			val local_var_3_0: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
			local_var_3_0.dispose()

		}
		if (kotlin.Any().equals(kotlin.Any())) {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.isDisposed()

		} else {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.isDisposed()

		}
		param0.method1(io.reactivex.subscribers.TestSubscriber())

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.hasSubscription()

	}

	fun method2(
		param0: com.android.example.module18_module1_module24_module1.ClassAAA,
		param1: com.android.example.module18_module1_module24_module1.ClassAAB,
		param2: com.android.example.module18_module1_module24_module3.ClassAAE
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.onError(kotlin.Throwable())

		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.requestMore(4425341672829664116)

	}
}
